src/command_line.rs

```diff
@@ -269,6 +269,7 @@ impl Config {
             Arg::Long("newsonupgrade") => self.news_on_upgrade = true,
             Arg::Long("nonewsonupgrade") => self.news_on_upgrade = false,
             Arg::Long("comments") => self.comments = true,
+            Arg::Long("ssh") | Arg::Short('e') => self.ssh = true,
```

Comment:
e -> s

Reply:
(so is `h`)

Reply:
Right you need to move 's' to it's own line and set both. see how 'c' is.

Reply:
Yeah that's how the code works ATM for the sake of lasiness

Reply:
Yeah, I realised that pacman's flags is a bit unusual in that it doesn't use typical subcommands.